dev-install.sh

```diff
@@ -52,8 +52,8 @@ echo -n "ipywidgets"
 pip install -v -e .
 
 if test "$skip_jupyter_lab" != yes; then
-    jupyter labextension link ./packages/base
-    jupyter labextension link ./packages/controls
-    jupyter labextension link ./packages/output
-    jupyter labextension install ./packages/jupyterlab-manager
+    jupyter labextension link ./packages/base --no-build
+    jupyter labextension link ./packages/controls --no-build
+    jupyter labextension link ./packages/output --no-build
+    jupyter labextension install ./packages/jupyterlab-manager --minimize=False
```

Comment:
Can you elaborate what those options will do?

Reply:
The first three `--no-build` prevent us rebuilding JupyterLab three unnecessary times. The last option builds JupyterLab, but with js minimization off for easier debugging.

Reply:
Thank you for clarification!

Reply:
@vidartf clarified that it *says* it is building a minimized build, but it's actually not.